scripts/builtin/img_mirror_linearized.dml

```diff
@@ -0,0 +1,82 @@
+#-------------------------------------------------------------
```

Comment:
You might want to separate the PRs completely to not include mirror inside this PR.